test/fixtures/utf8-skip-escape-characters.json

```diff
@@ -0,0 +1,61 @@
+{
+    "charset": "utf-8",
+    "headers": {
+        "Project-Id-Version": "gettext-parser",
+        "Report-Msgid-Bugs-To": "andris@node.ee",
+        "POT-Creation-Date": "2012-05-18 14:28:00+03:00",
+        "PO-Revision-Date": "2012-05-18 14:37+0300",
+        "Last-Translator": "Andris Reinman <andris@kreata.ee>",
+        "Language-Team": "gettext-parser <andris@node.ee>",
+        "Language": "",
+        "Content-Type": "text/plain; charset=utf-8",
+        "Content-Transfer-Encoding": "8bit",
+        "Plural-Forms": "nplurals=2; plural=(n!=1);",
+        "mime-version": "1.0",
+        "x-poedit-language": "Estonian",
+        "x-poedit-country": "ESTONIA",
+        "x-poedit-sourcecharset": "utf-8"
+    },
+    "translations": {
+        "": {
+            "": {
+                "msgid": "",
+                "comments": {
+                    "translator": "gettext-parser test file.\nCopyright (C) 2012 Andris Reinman\nThis file is distributed under the same license as the gettext-parser package.\nANDRIS REINMAN <andris@node.ee>, 2012.\n"
+                },
+                "msgstr": [
+                    "Project-Id-Version: gettext-parser\nReport-Msgid-Bugs-To: andris@node.ee\nPOT-Creation-Date: 2012-05-18 14:28:00+03:00\nPO-Revision-Date: 2012-05-18 14:37+0300\nLast-Translator: Andris Reinman <andris@kreata.ee>\nLanguage-Team: gettext-parser <andris@node.ee>\nLanguage: \nContent-Type: text/plain; charset=utf-8\nContent-Transfer-Encoding: 8bit\nPlural-Forms: nplurals=2; plural=(n!=1);\nmime-version: 1.0\nx-poedit-language: Estonian\nx-poedit-country: ESTONIA\nx-poedit-sourcecharset: utf-8\n"
+                ]
+            },
+            "o2-1": {
```

Comment:
I think you can remove this one too? As the `o3-õäöü` msgid below is basically the no-escape example, right?

Reply:
This one is needed due to line breaks in it. Good to test that